tests/test_sessions.py

```diff
@@ -108,6 +109,7 @@ def test_properties(self):
         assert session.bin is runner.venv.bin_paths[0]
         assert session.python is runner.func.python
         assert session.invoked_from is runner.global_config.invoked_from
+        assert session.cache_dir == pathlib.Path(runner.envdir).joinpath(".shared")
```

Comment:
Just one last thing, you'll need to import `pathlib` in the test file for this to work.

Reply:
Oops, you're right.

Reply:
Actually, that line could be relaced by

```python
        assert session.cache_dir == Path(runner.envdir).joinpath(".shared")
```

(`Path` has been imported before, but `pathlib` not).

Reply:
Yeah that would work